pvmismatch/contrib/xlsio/example_workflow/example_workflow.py

```diff
@@ -0,0 +1,56 @@
+import pandas as pd
```

Comment:
@adambgnr 
to your point about Pandas as a dependency, you could add a try except clause here and warn user to install pandas before proceeding.

Reply:
@chetan201 
Thanks, I have made the changes accordingly: 89cd532526f222d7788527e2a6c4dab2dd9bebc3